starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,41 @@
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    width:1024px;
+    margin: 0 auto;
+    padding-bottom:100px;
+}
+
+#hero {
```

Comment:
The page looks really good. 🚀 
In the next exercise try to practice using the tags of HTML5. 
(only to get some practice 💅🏼)
I really like to use scrimba to study new topics then I will recommend this link: https://scrimba.com/p/pZaVfV/cWNRbCR